helix-core/src/indent.rs

```diff
@@ -452,6 +452,7 @@ where
                 file_types: vec!["rs".to_string()],
                 shebangs: vec![],
                 language_id: "Rust".to_string(),
+                display_name: "Rust".to_string(),
```

Comment:
Why do we want display name? Just for estatic reasons?

Reply:
Yeah pretty much, it looks better in the docs (`c-sharp` vs `C#`) and we can also reuse it for #976.